src/components/modals/WcSmartContractModal.tsx

```diff
@@ -114,13 +104,8 @@ export const WcSmartContractModal = (props: Props) => {
         <IconTile title={lstrings.wc_smartcontract_dapp} iconUri={icon}>
           <EdgeText>{dAppName}</EdgeText>
         </IconTile>
-        {!zeroString(networkFeeCrypto) && (
-          <CryptoFiatAmountTile
-            title={lstrings.wc_smartcontract_network_fee}
-            nativeCryptoAmount={networkFeeCrypto}
-            denomination={feeDenom}
-            walletId={wallet.id}
-          />
+        {!zeroString(networkFee) && (
```

Comment:
We are avoiding the `&&` for rendering, since we want to enable strict boolean linting. So `zeroString(networkFee) ? null : <CryptoFiatAmountTile ... />`